big_o/big_o.py

```diff
@@ -173,3 +173,52 @@ def big_o(func, data_generator,
         fitted['times'] = time
 
     return best, fitted
+
+def print_report(best, others):
+    """ Prints the detailed summary of the calculated reports
```

Comment:
"Creates a human-readable report of the output of the `big_o` function."

Reply:
Done returns string now